Update IDE GA banner (#2339)
dbt-labs/docs.getdbt.com: website/snippets/ide-ga-banner.md

:::info 📌

> The new and refreshed Cloud IDE is open to General Availability. The new Cloud IDE includes performance upgrades, ergonomics improvements, and some delightful enhancements!

Reviewer: ```suggestion
The new and refreshed Cloud IDE is now GA. It includes performance upgrades, ergonomics improvements, and some delightful enhancements!
```
